utils/generator/test/runner.py

```diff
@@ -0,0 +1,41 @@
+"""
+Main entry point to run all tests
+"""
+import sys
+from pathlib import Path
+from unittest import TestLoader, TestSuite, TextTestRunner
+
+PATH = Path(__file__).absolute()
+
+sys.path.append(PATH.parents[1].joinpath('rpc_spec/InterfaceParser').as_posix())
+sys.path.append(PATH.parents[1].as_posix())
+
+try:
+    from test_order import TestOrder
+    from test_enums import TestEnumsProducer
+    from test_functions import TestFunctionsProducer
+    from test_structs import TestStructsProducer
+    from test_code_format_and_quality import CodeFormatAndQuality
+except ImportError as message:
+    print('{}. probably you did not initialize submodule'.format(message))
+    sys.exit(1)
+
+
+def main():
+    """
+    Main entry point to run all tests
+    """
+
+    suite = TestSuite()
+    suite.addTests(TestLoader().loadTestsFromTestCase(TestOrder))
+    suite.addTests(TestLoader().loadTestsFromTestCase(TestFunctionsProducer))
+    suite.addTests(TestLoader().loadTestsFromTestCase(TestEnumsProducer))
+    suite.addTests(TestLoader().loadTestsFromTestCase(TestStructsProducer))
+    suite.addTests(TestLoader().loadTestsFromTestCase(CodeFormatAndQuality))
+
+    runner = TextTestRunner(verbosity=2)
+    runner.run(suite)
+
+
+if __name__ == '__main__':
```

Comment:
Would it be possible to trigger the tests on every push using Git Workflow CI? similar to how we do that for Android, JavaSE, and JavaEE tests here:
https://github.com/smartdevicelink/sdl_java_suite/blob/develop/.github/workflows/android.yml

Reply:
I will add this to github actions file.

Reply:
Added tests to github actions, build works fine